libs/designsystem/form-field/src/input-counter/input-counter.component.spec.ts

```diff
@@ -82,7 +82,7 @@ describe('InputCounterComponent', () => {
 
   describe('when configured with listenTo = input', () => {
     describe('and input does not have initial value and maxlength', () => {
-      const input = new InputComponent();
+      const input = new InputComponent(null);
```

Comment:
The input component requires an argument after I added a constructor to the component.